### DISCUSSION TITLE
Authenticated user is not removed upon uninstallation on iOS and Android and then reinstallation (not update in place)

### DISCUSSION THREAD

**Discussion by bangonkali** [capawesome-team/capacitor-firebase#485] [Q&A] [answered]
We transitioned to `@capacitor-firebase/authentication` for an Ionic React Capacitor project. For some reason we have uninstalled the app but the Logged In state does not disappear /clear after reinstallation of the app. Is this a feature and by design? Is there a way to turn this off?

**Comment by robingenz** [comment, marked as answer]
TLDR: This is a feature.

See https://stackoverflow.com/a/55611847/6731412.

**Comment by bangonkali** [comment]
Hi @robingenz is there a way to turn this off?

**Reply by robingenz** [reply]
No, as far as i know. You could call `signOut()` directly when you start the app for the first time.
